docs/background.md

```diff
@@ -0,0 +1,95 @@
+# Background
+
+We capture motivational context for the creation of the in-toto Attestation
+Framework here.
+
+## Motivation
+
+This project was initiated with two main motivations:
+
+1.  Support use cases where the prior in-toto link schema was a poor fit. For
+    example, test steps and vulnerability scans are not about "producing" a new
+    artifact so they are awkward to represent in the current format.
+2.  Support interoperability with [Binary Authorization], which will support
+    the agreed-upon format once finalized. This way we have a single ecosystem
+    of software supply chain security.
```

Comment:
Do we know that this is true?

Reply:
this part of the doc was [originally authored](https://github.com/in-toto/attestation/commit/9c61d1825f492dca00d6ee12f0f21ae695df3ee0#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R627) by @MarkLodato, perhaps he can comment?

cc @TomHennen

Reply:
I'd suggest changing 'will' to 'can' just to cover our bases.

Reply:
Done, thanks!